Source/Krypton Components/Krypton.Ribbon/Controller/GalleryButtonController.cs

```diff
@@ -116,7 +116,7 @@ public virtual bool MouseDown(Control c, Point pt, MouseButtons button)
                 if (Target.Enabled)
                 {
                     OnClick(new MouseEventArgs(MouseButtons.Left, 1, pt.X, pt.Y, 0));
-                    _repeatTimer.Start();
+                    _repeatTimer?.Start();
```

Comment:
If this is doing a `start`, then should it not have previously checked to see if timer should be created ? And create if Null ?

Reply:
Naah, The Constructor allows for the timer to be used or not (boolean param).
So it's either a valid reference or null.